pkg/gateway/model_build_targets.go

```diff
@@ -189,7 +206,7 @@ type latticeTargetsModelBuildTask struct {
 	tgName         string
 	tgNamespace    string
 	routename      string
-	port           int64
+	port           int32
 	latticeTargets *latticemodel.Targets
```

Comment:
Could you change it's name to routeBackendRefPort ?

Reply:
Fixed, but backendRef already implies route so I prefer backendRefPort.